site/examples/gallery.md

```diff
@@ -23,5 +23,8 @@ Here we list great examples of Vega-Lite visualizations that were created by the
 * [Grouped Bar Chart by @churtado](https://bl.ocks.org/domoritz/f5abc519dd990bfcbc3f20f634658364)
 * [Multi Line Highlight by @amitkaps](https://bl.ocks.org/amitkaps/fe4238e716db53930b2f1a70d3401701)
 * [Slope graph by @g3o2](https://bl.ocks.org/g3o2/a6c539eacfb0b99eaf01e4f20b9f2897)
-* [Scatter Nearest Rule](https://bl.ocks.org/amitkaps/abfa7157d4366cc43cbbba55353d35d8)
-* [Scatter Brush Rule](https://bl.ocks.org/amitkaps/a484b94a7e1e0705c5ec865ba31f463c)
+* [Scatter Nearest Rule by @amitkaps](https://bl.ocks.org/amitkaps/abfa7157d4366cc43cbbba55353d35d8)
+* [Scatter Brush Rule by @amitkaps](https://bl.ocks.org/amitkaps/a484b94a7e1e0705c5ec865ba31f463c)
+* [Unit Chart Rectangular by @amitkaps](https://bl.ocks.org/amitkaps/d6648bd8ddb1c1e3706d7530126d1e2b)
+* [Unit Chart Stacked by @amitkaps](https://bl.ocks.org/amitkaps/cdc7dacd8f7d9f2a9cff4b10d3279b86)
+* [Unit Chart Small Multiple by @amitkaps](https://bl.ocks.org/amitkaps/67bd6dcb2af300a2b76f1e2351c1afdc)
```

Comment:
I wonder whether you could reuse the code from https://bl.ocks.org/amitkaps/d6648bd8ddb1c1e3706d7530126d1e2b and not add the offset manually if you used `facet`? Can you give it a try?